firehose/Dockerfile

```diff
@@ -32,5 +32,17 @@ COPY --from=go_build /app/firehose-core/firecore /app/firecore
 COPY --from=go_build /app/firehose-core/devel/standard/bootstrap.sh /app/start.sh
 RUN chmod +x /app/start.sh
 
-# Additional setup if needed
-ENTRYPOINT [ "/app/start.sh" ]
+# Set environment variables for port configuration
+ENV FIREHOSE_PORT_1=10015
+ENV FIREHOSE_PORT_2=10016
+ENV FIREHOSE_PORT_3=10017
+# Expose ports
+EXPOSE $FIREHOSE_PORT_1
+EXPOSE $FIREHOSE_PORT_2
+EXPOSE $FIREHOSE_PORT_3
+
+# Set volume
+VOLUME ["/data/storage_dir"]
+
+ENTRYPOINT ["/app/start.sh"]
+CMD ["$CHAIN_ID"]
```

Comment:
does this mean the ENV for CHAIN_ID must be set before this image starts?

Reply:
Yes. Pass it as env or argument.